dev/docker/trino/Dockerfile

```diff
@@ -45,11 +45,19 @@ RUN rm -rf /usr/lib/trino/plugin/accumulo  \
     && rm -rf /usr/lib/trino/plugin/redis \
     && rm -rf /usr/lib/trino/plugin/session-property-managers \
     && rm -rf /usr/lib/trino/plugin/teradata-functions
-
 COPY --chown=trino:trino packages/trino/conf /etc/trino
 COPY --chown=trino:trino packages/gravitino-trino-connector/mysql-connector-java-8.0.27.jar /usr/lib/trino/plugin/iceberg/
 COPY --chown=trino:trino packages/gravitino-trino-connector/postgresql-42.7.0.jar /usr/lib/trino/plugin/iceberg/
-COPY --chown=trino:trino packages/gravitino-trino-connector /usr/lib/trino/plugin/gravitino
+COPY --chown=trino:trino packages/gravitino-trino-connector /tmp/
+
+ARG IMAGE_NAME
+RUN if [ "$IMAGE_NAME" = "datastrato/trino" ] ; then \
+    mkdir /usr/lib/trino/plugin/gravitino; \
+    cp -rf /tmp/*.jar /usr/lib/trino/plugin/gravitino/; \
```

Comment:
I think would be better to change to `cp -r /tmp/gravitino-trino-connector /usr/lib/trino/plugin/gravitino`?

Reply:
I change command
`COPY --chown=trino:trino packages/gravitino-trino-connector /tmp/` to `COPY --chown=trino:trino packages/gravitino-trino-connector /tmp/gravitino-trino-connector`

and 
`cp -rf /tmp/*.jar /usr/lib/trino/plugin/gravitino/;` to `cp -rf /tmp/gravitino-trino-connector/*.jar /usr/lib/trino/plugin/gravitino/`